testdata/project-v4-multigroup-with-deploy-image/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Build the manager binary
+# [Choice] Go version (use -bullseye variants on local arm64/Apple Silicon): 
+# 1, 1.19, 1.18, 1-bullseye, 1.19-bullseye, 1.18-bullseye, 1-buster, 1.19-buster, 1.18-buster
+ARG TAG=1.22
+FROM golang:${TAG}
+
+COPY post-install.sh .
+
+# Install dependencies
+RUN curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl" && \
+    install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+
+RUN go install sigs.k8s.io/kind@v0.23.0 && \
+    curl -o kubebuilder \
+      -L https://github.com/kubernetes-sigs/kubebuilder/releases/latest/download/kubebuilder_linux_amd64 && \
+    chmod +x kubebuilder && \
+    mv kubebuilder /usr/local/bin/ && \
+    chmod +x post-install.sh
```

Comment:
we do not need the dockerfile
We can use an image: https://containers.dev/implementors/json_reference/#image-specific
```
{
  "name": "Kubebuilder Dev Container",
  "image": "ubuntu:20.04", // Base Ubuntu image
  "features": {
    "docker-in-docker": {
      "version": "latest"
    }
  },
  "postCreateCommand": [
    "apt-get update && apt-get install -y curl apt-transport-https ca-certificates gnupg-agent software-properties-common lsb-release",
    "curl -fsSL https://get.docker.com -o get-docker.sh",
    "sh get-docker.sh",
    "usermod -aG docker vscode",
    "KIND_VERSION=v0.15.0",
    "curl -Lo /usr/local/bin/kind https://kind.sigs.k8s.io/dl/${KIND_VERSION}/kind-linux-amd64",
    "chmod +x /usr/local/bin/kind",
    "KUBEBUILDER_VERSION=2.4.1",
    "curl -Lo kubebuilder.tar.gz https://dl.kubebuilder.io/kubebuilder/${KUBEBUILDER_VERSION}/kubebuilder_${KUBEBUILDER_VERSION}_linux_amd64.tar.gz",
    "tar -zxvf kubebuilder.tar.gz -C /usr/local/",
    "rm kubebuilder.tar.gz",
    "ln -s /usr/local/kubebuilder/bin/kubebuilder /usr/local/bin/kubebuilder",
    "kind version",
    "kubebuilder version",
    "docker --version",
    "apt-get clean"
  ],

```